### PR TITLE
DOCS-11893 updating status

### PR DESCRIPTION
**Pull request by IsaacEldridge**

### CODE REVIEW

**Review by mauro-dentoni — APPROVED** [review]

File: modules/ROOT/pages/status-updates.adoc [hunk 1]
```diff
@@ -4,15 +4,19 @@ When working with APIs and the unified schema, status updates let you know when
 
 == Status: Updating
 
-You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema will enter an “updating” state, and the status will indicate *Updating*. This status update is visible to anyone in the business group with permissions for the environment.
+You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema enters an “updating” state, and the status indicates the progress and queued updates. This status update is visible to anyone in the business group with permissions for the environment.
```
Comment (kbostrom) [inline modules/ROOT/pages/status-updates.adoc:7]: Do we need the first "and apply changes"? Does "add to and update" not imply that? Your choice, of course:
```suggestion
You can add to and update the unified schema at any time. After you apply changes, the unified schema enters an `updating` state during which you can view the status of in-progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```

File: modules/ROOT/pages/status-updates.adoc [hunk 2]
```diff
@@ -4,15 +4,19 @@ When working with APIs and the unified schema, status updates let you know when
...
 
-image::status_updating.png[Status indicator shows adding API schema to unified schema,75%,75%]
+image::datagraph-status-update-in-process.png[Status indicator shows adding API schema to unified schema,65%,65%]
 
 When the unified schema is updating, you can view the new changes in the schema, make additional changes, and apply new changes. However, these changes aren’t available to query until the update is complete.
 
-When the update is complete, the status will change to *Up to date*.
+As queued updates finish, the status changes to finished.
```
Comment (kbostrom) [inline modules/ROOT/pages/status-updates.adoc:13]: As with "updating" in Line 7, is this a syntax that the user sees?
Use mono for syntax display; bold for UI labels.

Comment (kbostrom) [reply modules/ROOT/pages/status-updates.adoc:13]: See also Line 17 below.
I've edited for syntax presentation but you know best: Lines 7, 13, and 17.

**Review by kbostrom — DISMISSED** [review]
PR CE

File: modules/ROOT/pages/status-updates.adoc [hunk 3]
```diff
@@ -4,15 +4,19 @@ When working with APIs and the unified schema, status updates let you know when
 
 == Status: Updating
 
-You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema will enter an “updating” state, and the status will indicate *Updating*. This status update is visible to anyone in the business group with permissions for the environment.
+You can add to and update the unified schema at any time. After you apply changes, the states reads *Updating* and indicates the progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```
Comment (kbostrom) [inline modules/ROOT/pages/status-updates.adoc:7]:
```suggestion
You can add to and update the unified schema at any time. After you apply changes, the status reads *Updating* and indicates the progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```

File: modules/ROOT/pages/status-updates.adoc [hunk 4]
```diff
@@ -4,15 +4,19 @@ When working with APIs and the unified schema, status updates let you know when
...
 
 When the unified schema is updating, you can view the new changes in the schema, make additional changes, and apply new changes. However, these changes aren’t available to query until the update is complete.
 
-When the update is complete, the status will change to *Up to date*.
+As queued updates finish, the status changes to *finished*.
 
-image::status_uptodate.png[Status indicator shows unified schema is up to date,75%,75%]
+image::datagraph-status-update-finished.png[Status indicator shows an update is finished,65%,65%]
+
+When all updates complete, the status will change to *Up to date*.
```
Comment (kbostrom) [inline modules/ROOT/pages/status-updates.adoc:17]:
```suggestion
When all updates are complete, the status changes to *Up to date*.
```

**Review by kbostrom — APPROVED** [review]
PR RA

**Pull request merged**